src/get-own-property-symbols.js

```diff
@@ -266,7 +277,7 @@
             ;
             return typeof tst === 'undefined' ? str : ('[object ' + tst + ']');
           };
-          dP(ObjectProto, 'toString', descriptor);
+          dP(/** @type {!Object} */(ObjectProto), 'toString', descriptor);
```

Comment:
I prefer not to use jsdoc in my projects.

Reply:
Typecheckers (not just Closure) will otherwise complain, as the call would be with an incorrect value. Do you have a suggested replacement for any JSDoc statement?

Reply:
Your type checker shouldn’t be checking code you didn’t write.

The way Flow and TS handle this is with external packages that define the types for a package’s interface. Does Closure Compiler not have a similar capability?

Reply:
Afaik, Closure implements optimizations that require full knowledge of source code, including dependencies. TS and Flow only typecheck and do not perform such optimizations. Closure therefore can not optimize based on this package as well as it potentially could, whereas TS and Flow-typed version would be okay.

Reply:
your explanation makes sense, but that's why "advanced mode" is terrible: because it forces code changes on your code (and apparently, on your entire dependency tree).